### PR TITLE
Correct inclusion of `www` prefix to prevent redirects

### PR DESCRIPTION
**Pull request by adunning**
This consistently removes the `www.` prefix from social links. There doesn't seem to have been a particular pattern before: Twitter, for example, certainly prefers the prefix to be removed.

### CODE REVIEW

**Comment by adunning** [comment]
Think I have all these right now.

File: _includes/social.html [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+  {%- if site.instagram_username -%}<li><a href="https://www.instagram.com/{{ site.instagram_username| cgi_escape | escape }}"><svg class="svg-icon"><use xlink:href="{{ '/assets/minima-social-icons.svg#instagram' | relative_url }}"></use></svg> <span class="username">{{ site.instagram_username| escape }}</span></a></li>{%- endif -%}
+  {%- if site.linkedin_username -%}<li><a href="https://www.linkedin.com/in/{{ site.linkedin_username| cgi_escape | escape }}"><svg class="svg-icon"><use xlink:href="{{ '/assets/minima-social-icons.svg#linkedin' | relative_url }}"></use></svg> <span class="username">{{ site.linkedin_username| escape }}</span></a></li>{%- endif -%}
   {%- if site.pinterest_username -%}<li><a href="https://pinterest.com/{{ site.pinterest_username| cgi_escape | escape }}"><svg class="svg-icon"><use xlink:href="{{ '/assets/minima-social-icons.svg#pinterest' | relative_url }}"></use></svg> <span class="username">{{ site.pinterest_username| escape }}</span></a></li>{%- endif -%}
```
Comment (DirtyF) [inline _includes/social.html]: still one missing...

```http
http -h https://pinterest.com/
HTTP/1.1 301 Moved Permanently
Date: Tue, 24 Apr 2018 21:31:29 GMT
Location: https://www.pinterest.com/
```

Comment (adunning) [reply _includes/social.html]: Noticed that earlier; not sure why the older commit was still showing for you.

**Comment by adunning** [comment]
Thanks for taking care of it!

**Comment by ddebernardy** [comment]
For some reason Twitter continues to boast a www.

**Pull request merged**